homeassistant/components/homekit_controller/cover.py

```diff
@@ -214,34 +214,42 @@ def is_vertical_tilt(self) -> bool:
     @property
     def current_cover_tilt_position(self) -> int | None:
         """Return current position of cover tilt."""
-        tilt_position = self.service.value(CharacteristicsTypes.VERTICAL_TILT_CURRENT)
-        if not tilt_position:
-            tilt_position = self.service.value(
-                CharacteristicsTypes.HORIZONTAL_TILT_CURRENT
+        tilt_position = None
+
+        def _get_tilt_values(
+            characteristic: str,
+        ) -> tuple[Any, int | float | None, int | float | None]:
+            """Fixture min and max values and tilt position retrieval."""
+            return (
+                self.service.value(characteristic),
+                self.service[characteristic].minValue,
+                self.service[characteristic].maxValue,
             )
```

Comment:
Get rid of this helper. Just get the char out of self.service once and do char.value, char.minValue and char.maxValue.

Reply:
amended